.travis.yml

```diff
@@ -140,7 +129,6 @@ script:
   - $make CFG_TEE_CORE_LOG_LEVEL=0 CFG_TEE_CORE_DEBUG=y DEBUG=1
   - $make CFG_TEE_CORE_LOG_LEVEL=0 CFG_TEE_CORE_DEBUG=n DEBUG=0
   - $make CFG_TEE_CORE_LOG_LEVEL=0 CFG_TEE_CORE_DEBUG=n CFG_TEE_TA_LOG_LEVEL=0 DEBUG=0
-  - $make CFG_TEE_CORE_MALLOC_DEBUG=y
```

Comment:
This was this only case the script is building bget with the debug support.
Should we really remove it ? If you feel it's not that useful, maybe comment it until current automated test issue is solved.